controllers/events_controller.js

```diff
@@ -399,11 +419,17 @@ module.exports.addEvent = async (req, res) => {
 		venue,
 		isRegistrationOpened,
 		isRegistrationRequired,
-		code
+		code,
+		maxRegister
 	});
 
-	if (req.files && req.files.length !== 0 ) {
-		event.image = req.files[0].location;
+	if (req.files && req.files.length !== 0) {
+		let file = req.files[0];
+		let key = getImageKey(req.originalUrl);
+		let uploaded = await uploadImage(file, key);
+		if (uploaded) {
```

Comment:
Will the variable needs to be checked only for true/false values or should we also check some other parameters? What is it returning specifically?

Reply:
if image is uploaded, it gives the URL, else undefined